components/cd-flow/cd-flow.css

```diff
@@ -0,0 +1,25 @@
+/* https://piccalil.li/cube-css */
+
+/* CUBE CSS is a CSS methodology */
+/* orientated towards simplicity, pragmatism and consistency */
+/* Composition Utility Block Exception */
+
+.cd-flow > * + * {
+  margin-top: var(--cd-flow-space, 1rem);
+}
+
+.cd-flow > .visually-hidden + * {
+  margin-top: 0;
+}
+
+[dir='ltr'] .cd-bumper > * + * {
+  padding-left: var(--cd-bumper-space, 1rem);
+}
```

Comment:
it's unclear to me what `cd-bumper` accomplishes. I see it used in `cd-author` and `cd-byline` but Flow is about vertical rhythm and Bumper seems to be about text directionality.

The demo page understandably doesn't have `cd-flow` as a dedicated component, so perhaps the README could include a few words about how bumper is intended to be used.

Reply:
Oops, ya, I didn't realise the README was essentially blank.
The `cd-bumper` is an attempt to control horizontal spacing. I bulked this in with `cd-flow` because I'm not sure how to solve the issue of consistent horizontal spacing, and it's a similar problem to vertical rhythm.
I'm unsure about whether this is the way to do it, and so I maintain it doesn't warrant its own `cd-bumper` component as I still might remove it altogether.
Also, `cd-flow` could be renamed to `cd-cubecss` or something if I need it to accommodate more rules.